src/app/components/Image/BlurredBackground.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from './index.styles';
+
+type BlurredBackgroundProps = {
+  src?: string;
+};
+
+const BlurredBackground = ({ src }: BlurredBackgroundProps) => {
```

Comment:
Rather than passing `isLite` around as props, we could just get it from `RequestContext` in there directly, unless you're wanting to keep this component more "pure"?